packages/beacon-node/src/execution/engine/http.ts

```diff
@@ -417,6 +437,35 @@ export class ExecutionEngineHttp implements IExecutionEngine {
     return response.map(deserializeExecutionPayloadBody);
   }
 
+  getState(): ExecutionEngineState {
+    return this.state;
+  }
+
+  async getClientVersion(clientVersion: ClientVersion): Promise<ClientVersion[]> {
+    const method = "engine_getClientVersionV1";
+
+    const response = await this.rpc.fetchWithRetries<
+      EngineApiRpcReturnTypes[typeof method],
+      EngineApiRpcParamTypes[typeof method]
+    >({method, params: [clientVersion]});
+
+    const clientVersions = response.map((cv) => {
+      const code = cv.code in ClientCode ? ClientCode[cv.code as keyof typeof ClientCode] : ClientCode.XX;
+      return {code, name: cv.name, version: cv.version, commit: cv.commit};
+    });
+
+    if (clientVersions.length > 0) {
+      this.clientVersion = clientVersions[0];
+      this.logger.info("Execution client version is updated", this.clientVersion);
```

Comment:
this does not look like a info log to me, rather debug or why does the user need to know about this internal detail?